constraint/go.mod

```diff
@@ -1,22 +1,20 @@
 module github.com/open-policy-agent/frameworks/constraint
 
-go 1.22.0
+go 1.22.7
 
-toolchain go1.22.5
+toolchain go1.23.5
 
 require (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
-	github.com/golang/glog v1.2.1
+	github.com/golang/glog v1.2.2
 	github.com/google/go-cmp v0.6.0
 	github.com/onsi/gomega v1.33.1
-	github.com/open-policy-agent/opa v0.68.0
+	github.com/open-policy-agent/opa v1.0.1
```

Comment:
OPA 1.1.0 is out - https://github.com/orgs/open-policy-agent/discussions/654, let's update to latest version if no one has any objections. @open-policy-agent/gatekeeper-maintainers

Reply:
updated to 1.1.0